cmd/kubeadm/app/phases/apiconfig/clusterroles.go

```diff
@@ -19,10 +19,11 @@ package apiconfig
 import (
 	"fmt"
 
+	"k8s.io/api/core/v1"
+	rbac "k8s.io/api/rbac/v1beta1"
+rbachelper "k8s.io/kubernetes/pkg/apis/rbac/v1beta1"
```

Comment:
Here's [a link to the comment](https://github.com/kubernetes/kubernetes/pull/44784#discussion_r121678895) against file _staging/src/k8s.io/api/apps/v1beta1/register.go_, which GitHub hides because the commit is outdated.